recipes/dioptas/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "Dioptas" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "5911464f2141f19720d8b987796d86344602737c9e4c34309d0b0dfd2b32d90b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Dioptas/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true # [win and py27]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  entry_points:
+    - dioptas = dioptas:main
+
+app:
+  entry: dioptas
+  type: desk
+  summary: "Dioptas"
+  icon: icon.png
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}  
+
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy >=1.11
+    - scipy
+    - pyqt
+    - pyqtgraph
+    - lmfit
+    - fabio
+    - pyfai
+    - pycifrw
+    - future
+    - scikit-image
+    - qtpy
+
+  run:
+    - python
+    - numpy >=1.11
```

Comment:
are you building against ``numpy`` or this is just a dependency of your python project?

Reply:
This gives the question whether  actually need packages other than:

python
setuptools 
numpy >= 1.11

for the host. Because the others are rather needed for running Dioptas but not for building the package. is this correct?

Reply:
You can use this
```
- {{ pin_compatbile('numpy') }}
```

Please, see
https://conda-forge.org/docs/meta.html#building-against-numpy